dbt_subprojects/solana/models/bitget_wallet/bgw_solana_swap.sql

```diff
@@ -0,0 +1,84 @@
+{{
+    config(
+
+        schema = 'bgw_solana',
+        alias = 'swap',
+        partition_by = ['block_date'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        post_hook='{{ expose_spells(\'["solana"]\',
+                                "project",
+                                "bgw",
+                                \'["clyde.ren"]\') }}'
```

Comment:
```suggestion
```

Reply:
proactively removing this post hook, assuming this spell will eventually feed into another? in that case, we don't always display the intermediate tables in the data explorer.